talisker/util.py

```diff
@@ -22,6 +22,9 @@
 import pip
 from builtins import *  # noqa
 
+import functools
+
+import werkzeug.local
```

Comment:
werkzeug is a library, with some wsgi library stuff in it. werkzeug.local is a thread/greenlet context, independent of anything wsgi. Using talisker with celery already requires werkzeug, as its a core talisker dependency.

Changing this would mean moving the wsgi parts of talisker to an optional dependency, and vendoring werkzeug.local functionality. Or publishing a whole separate talisker_celery package or similar. Both of which have downsides, and are beyond this MP